artifact-manifest-spec.md

```diff
@@ -0,0 +1,77 @@
+
+- **`blobs`** *array of objects*
+
+    A collection of 0 or more blobs. The blobs array is analogous to [oci.image.manifest layers][oci-image-manifest-spec-layers], however unlike [image-manifest][oci-image-manifest-spec], the ordering of blobs is specific to the artifact type. Some artifacts may choose an overlay of files, while other artifact types may store indepdent collections of files.
+
+    - Each item in the array MUST be a [descriptor][descriptor], and MUST NOT refer to another `manifest` providing dependency closure.
+    - The max number of blobs is not defined, but MAY be limited by [distribution-spec][oci-distribution-spec] implementations.
+    - An encountered `descriptor.mediaType` that is unknown to the implementation MUST be ignored.
+
+- **`subjectManifest`** *descriptor*
+
+   A REQUIRED reference to an existing manifest. The artifact is said to be dependent upon the referenced `subjectManifest`.
+   - The item MUST be a [descriptor][descriptor] representing a manifest. Descriptors to blobs are not supported. The distribution-spec MUST return a `400` response code when `subjectManifest` is not found, and not a manifest.
+   - Future versions of the `oci.artifact.manifest` MAY support artifacts without a `subjectManifest`.
+
+- **`annotations`** *string-string map*
+
+    This OPTIONAL property contains arbitrary metadata for the image manifest.
+    This OPTIONAL property MUST use the [annotation rules](annotations.md#rules).
+
+    See [Pre-Defined Annotation Keys][annotations]
+
+## Push Validation
+
+Following the [distribution-spec push api](https://github.com/opencontainers/distribution-spec/blob/main/spec.md#push), all `blobs` *and* the `subjectManifest` descriptors SHOULD exist when pushed to a distribution instance.
+
+## Lifecycle Management
+
+For Phase 1, artifact types will be limited to reference types. A reference type is an artifact that doesn't have a lifecycle unto itself. A container image is said to have an independent lifecycle. A reference type, such as an SBoM or signature have a lifecycle tied to the `subjectManifest`. When the `subjectManifest` is deleted or marked for garbage collection, the defined artifact is subject to deletion as well. A distribution instance SHOULD delete, (refCount -1) the artifact when the `subjectManifest` is deleted.
+
+### Tagged `referenceTypes`
+
+As signatures and SBoMs are not considered independent artifact types, they SHOULD NOT have a tag, simplifying the lifecycle management. As the `subjectManifest` is marked for deletion (refCount=0), the `referenctType` is also marked for deletion (refCount -1). However, these artifacts MAY have tags as future versions of the artifact manifest MAY support independent types. 
```

Comment:
> My understanding was that the garbage collector will not be using the reference link in the manifest?

@nishakm see here

Reply:
And above, I guess.

Reply:
I think we all recognize adding any content added to registries needs enough info to manage its lifecycle. This includes GC on incomplete uploads or enough information for users to delete their content.

While registries implement GC and deletion management differently, we're just making sure we've provided the right level of information in the manifests to enable lifecycle management and set user expectations for consistency across registries. 

For phase 1, we're scoping this tightly so we can see how this evolves over time. For instance, we definitely want to support independent artifacts with `subjectManifest` references that have tags. This is the larger #37 effort that this will evolve into. We're just scoping independent artifacts out for now, as it's not critical to our phase 1 scope for the fall of 2021

Reply:
typo.. referenctType